### PR TITLE
fix accept transactions

### LINKED ISSUE #259: 2 save actions are needed for the "Accepts transfers" feature
The root of the problem is here: https://github.com/code4romania/ro-help/blob/develop/ro_help/hub/admin.py#L106
The ``ngo`` object does not point to an updated ``accounts QuerySet`` so if you turn on the flag for ``Accepts transfers`` and add a bank account, you will still get the message "To accept IBAN Transfers you need to add at least one account.", even though the bank account has been created. The flag will remain unset.

Add all details needed to support money transfer (CIF, CUI, Donations description), check "Accepts tranfers" and add the bank account details before saving.
image

The "To accept IBAN Transfers you need to add at least one account." notification appears
image

And "Accepts transfers" remains unchecked.
image

Trying again with the bank account already in place works:
image

The same problem occurs when you delete the bank account WITHOUT unchecking "Accepts transfers" - the flag remains checked until a subsequent save.

### PR DESCRIPTION
**Pull request by costibleotu**
Fixes #259

### CODE REVIEW

File: ro_help/hub/admin.py [hunk 1]
```diff
@@ -102,8 +101,17 @@ def get_readonly_fields(self, request, obj=None):
     @transaction.atomic
     def save_model(self, request, ngo, form, change):
         super().save_model(request, ngo, form, change)
+
         if ngo.accepts_transfer:
-            if not ngo.accounts.all():
+            no_accounts = request.POST.get("accounts-TOTAL_FORMS", 0)
```
Comment (vtemian) [inline ro_help/hub/admin.py:106]: Let's have a separate function that takes `request.POST` and `int(no_accounts)`.
```python
def has_accounts(self, request, no_accounts):
    for account_id in range(no_accounts):
        if f"accounts-{account_id}-DELETE" in request:
            continue

        if request.get(f"accounts-{account_id}-iban") and request.get(f"accounts-{account_id}-bank"):
            return True

    return False

File: ro_help/hub/admin.py [hunk 2]
```diff
@@ -102,8 +101,17 @@ def get_readonly_fields(self, request, obj=None):
...
+            has_accounts = False
+            for i in range(int(no_accounts)):
+                if not f"accounts-{i}-DELETE" in request.POST:
+                    if request.POST.get(f"accounts-{i}-iban") and request.POST.get(f"accounts-{i}-bank"):
+                        has_accounts = True
+                        break
+
+            if has_accounts is False:
```
Comment (vtemian) [inline ro_help/hub/admin.py:114]:
```python
if not self.has_accounts(request.POST, int(no_accounts))
```

File: ro_help/hub/admin.py (hunk 1 above)
Comment (vtemian) [reply ro_help/hub/admin.py:106]: Maybe `account_idx` or `idx` instead of `account_id`.

**Comment by costibleotu** [comment]
@vtemian 10x for feedback. pls check now

**Pull request merged**